chat-plugins/mafia-data.js

```diff
@@ -53,12 +55,25 @@ exports.alignments = Object.assign(Object.create(null), {
 		color: `#000`,
 		memo: [
 			`Factional Communication: If there are other Cult-aligned players, you may PM them during the game.`,
-			`Factional Concept: The Cult can recruit other players into the Cult more often than not.`,
+			`Factional Concept: The Cult can recruit other players into the Cult more often than not. Ask the Host for more information about how this works exactly.`,
 			`You are aligned with the <span style="color:#000;font-weight:bold">Cult</span>. You win when all players without a Cult wincon are eliminated and at least one Cult-aligned player is still alive (or nothing can prevent the same).`,
 		],
 		// TODO image for cult faction
 		image: `<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-villager.png"/>`,
 	},
+	cultafia: {
+		name: `Cultafia`,
+		plural: `Cultafia`,
+		id: `cultafia`,
+		color: `#600`,
+		memo: [
+			`Factional Communication: If there are other Cult-aligned players, you may PM them during the game.`,
```

Comment:
are cultafia separate from normal cults?

Reply:
Yes. (And you can't do Cult Mafia)